### PR TITLE
Correction of capital letters.

### PR DESCRIPTION
**Pull request by kekek2**

### CODE REVIEW

File: src/opnsense/mvc/app/controllers/OPNsense/TrafficShaper/forms/dialogPipe.xml [hunk 1]
```diff
@@ -1,31 +1,31 @@
...
         <id>pipe.bandwidth</id>
-        <label>bandwidth</label>
+        <label>Bandwidth</label>
         <type>text</type>
         <help>Total bandwidth for this pipe</help>
     </field>
     <field>
         <id>pipe.bandwidthMetric</id>
-        <label>bandwidth Metric</label>
+        <label>Bandwidth Metric</label>
```
Comment (fichtner) [inline src/opnsense/mvc/app/controllers/OPNsense/TrafficShaper/forms/dialogPipe.xml]: Can we agree on first letter uppercase, all else normal lowercase unless such things like "IP" or "IPsec" or "OpenVPN" or "CoDel" ?

File: src/opnsense/mvc/app/controllers/OPNsense/TrafficShaper/forms/dialogPipe.xml [hunk 2]
```diff
@@ -34,7 +34,7 @@
     </field>
     <field>
         <id>pipe.scheduler</id>
-        <label>Scheduler type</label>
+        <label>Scheduler Type</label>
```
Comment (fichtner) [inline src/opnsense/mvc/app/controllers/OPNsense/TrafficShaper/forms/dialogPipe.xml]: would mean this stays as it was before

File: src/opnsense/mvc/app/controllers/OPNsense/TrafficShaper/forms/dialogPipe.xml [hunk 3]
```diff
@@ -47,14 +47,14 @@
     </field>
     <field>
         <id>pipe.codel_target</id>
-        <label>(FQ-)CoDel target</label>
+        <label>(FQ-)CoDel Target</label>
```
Comment (fichtner) [inline src/opnsense/mvc/app/controllers/OPNsense/TrafficShaper/forms/dialogPipe.xml]: this stays, too

File: src/opnsense/mvc/app/controllers/OPNsense/TrafficShaper/forms/dialogPipe.xml [hunk 4]
```diff
@@ -47,14 +47,14 @@
...
         <type>text</type>
         <advanced>true</advanced>
         <help>Minimum acceptable persistent queue delay (in ms), leave empty for default</help>
     </field>
     <field>
         <id>pipe.codel_interval</id>
-        <label>(FQ-)CoDel interval</label>
+        <label>(FQ-)CoDel Interval</label>
```
Comment (fichtner) [inline src/opnsense/mvc/app/controllers/OPNsense/TrafficShaper/forms/dialogPipe.xml]: this too

File: src/opnsense/mvc/app/controllers/OPNsense/TrafficShaper/forms/dialogPipe.xml (hunk 1 above)
Comment (mimugmail) [reply src/opnsense/mvc/app/controllers/OPNsense/TrafficShaper/forms/dialogPipe.xml]: In Plugins all labels begin with upper first on all words I think

Comment (fichtner) [reply src/opnsense/mvc/app/controllers/OPNsense/TrafficShaper/forms/dialogPipe.xml]: looks mixed. we need to establish a baseline from core either way

Comment (fichtner) [reply src/opnsense/mvc/app/controllers/OPNsense/TrafficShaper/forms/dialogPipe.xml]: (and ideally avoid bouncing perfectly good translations)

File: src/opnsense/mvc/app/controllers/OPNsense/TrafficShaper/forms/dialogPipe.xml (hunk 4 above)
Comment (NOYB) [reply src/opnsense/mvc/app/controllers/OPNsense/TrafficShaper/forms/dialogPipe.xml]: System Information widget:
CPU Type
MBUF Usage

Services widget:
Gateway Monitoring Daemon
System Configuration Daemon
Users and Groups
Network Time Daemon
Secure Shell Daemon
Packet Filter

Would be nice if you guys would make up your mind.  ;)

**Comment by fichtner** [comment]
Bounces too many strings, no updates.

**Pull request closed without merging**